pages/datasets/[datasetid]/access/request/[requestid].vue

```diff
@@ -83,92 +87,188 @@ if (error.value) {
+                    Click HERE to download your SAS URI
+                    <Icon name="mdi:file-download-outline" class="ml-1" />
+                  </NuxtLink>
+                </n-card>
+
+                <p>
+                  Detailed instructions on AzCopy, its installation, and its use
+                  are available from the
+                  <NuxtLink
+                    to="https://learn.microsoft.com/en-us/azure/storage/common/storage-use-azcopy-v10?tabs=dnf"
+                    target="_blank"
+                  >
+                    Official Microsoft Documentation
+                  </NuxtLink>
+                  Below is a brief summary of steps you'll need to accomplish
+                  your download:
+                </p>
+
+                <ol>
+                  <li>
+                    Visit the Azcopy documentation page and
+                    <NuxtLink
+                      to="https://learn.microsoft.com/en-us/azure/storage/common/storage-use-azcopy-v10#download-azcopy"
+                      target="_blank"
+                    >
+                      download
+                    </NuxtLink>
+                    the compressed azcopy executable for your respective
+                    operating system. These executables are available as
+                    compressed zip files for Windows/Mac users and Tarball files
+                    for Linux users.
+                  </li>
 
-            <ol>
-              <li>
-                Download your SSH key using the link below, and save it in a
-                secure location. Make sure that the file permissions are set
-                read-only for your account.
+                  <li>
+                    Extract the contents of the compressed executable in a known
+                    location on your local machine's drive. - Windows and Mac
+                    users can use built-in zip file applications or use
+                    3rd-party applications like 7zip for this - For linux users,
+                    please see documentation for your specific distro for
+                    instructions on decompressing the provided tarball -
+                    Possible locations for this are: in your home directory; in
+                    your machine's C:/ or Root directory - For ease of use,
+                    consider adding the fully-qualified path to the AzCopy
+                    directory to your system PATH. If you choose not to do this
+                    you will need to execute any AzCopy commands from the
+                    directory where AzCopy is installed.
+                  </li>
+
+                  <li>
+                    Open a terminal or powershell instance and execute the
+                    following:
+                    <n-code
+                      code="azcopy copy 'C:\local\path' 'SAS URI' --recursive=true"
+                      language="powershell"
```

Comment:
This should be reversed 
```suggestion
                      code="azcopy copy 'SAS URI' 'C:\local\path' --recursive=true"
                      language="powershell"
```